core/build.sbt

```diff
@@ -452,7 +449,7 @@ def cloudflowModule(moduleID: String): Project = {
 lazy val commonSettings = Seq(
   organization := "com.lightbend.cloudflow",
   bintrayOrganization := Some("lightbend"),
-  bintrayRepository := altBintrayRepository.value.getOrElse("cloudflow"),
+  bintrayRepository := "cloudflow",
```

Comment:
Kept this for now as a reference if we needed it in the future, but I could remove it completely.